modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc

```diff
@@ -68,6 +68,18 @@ Successfully updated configuration. New configuration version is 2.
 
 NOTE: You can also change property values using the `rpk cluster config set` command, but this method does not display the current setting or the description.
 
+== Lock Properties at the Cluster Level
+
+In many instances, cluster administrators wish to lock certain properties in a way that individual topic settings cannot override them. If a property is locked then any attempts to edit that property at a topic level fail. For example, if you lock `internal_topic_replication_factor` at a value of 3, then any attempt to set `replication.factor` for a topic will yield an error.
+
+NOTE: No properties shipped with a locked setting. Property locking is a strictly opt-in feature.
+
+Enabling a property lock on a topic uses a straightforward syntax - you append `;locked` to the property value. For example, if you run `rpk cluster config edit` you might set `log_compression_type=producer;locked`. When viewing the property, you will see the appended keyword giving a quick identification of the presence of the lock. Any YAML generated via `rpk cluster config export` also includes the presence of the locked attribute.
+
+NOTE: You can only set a property to locked when providing a new or existing value for the property. You cannot set a property to locked with a default value unless you explicitly specify that value along with the locked attribute.
```

Comment:
Not sure I follow the second statement here--so if a property has a default value you cannot lock it in? I had to read this a few times and I am still not clear what the behavior is here. Can you clarify? Thx.

Reply:
That's correct, you can only lock it when setting it the the same or a new value. I'll try to rephrase this better.